views/layouts/layout-example.njk

```diff
@@ -1,31 +1,52 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8">
+{% extends "govuk/template.njk" %}
+
+{% set htmlClasses = 'no-js' %}
+{% set assetUrl = 'https://design-system.service.gov.uk/assets' %}
+
+{% block pageTitle %}{{ title }} – Example - GOV.UK Design System{% endblock %}
+
+{% block head %}
+  {% if not ENVIRONMENT or ENVIRONMENT != 'production' %}
     <meta name="robots" content="noindex, nofollow">
```

Comment:
This conditional is new right? What's the thinking behind it?

Reply:
@lfdebrux Good spot, thanks! That's an unintentional change, examples should always be not indexed. I'll amend the commit.